packages/webservice/app/src/main/resources/demo-configs/db/sql/DemoConnection.hjson

```diff
@@ -2,7 +2,7 @@
   dbconfigs: [
     {
       name: DemoConnection
-      url: jdbc:h2:mem:SalesDB;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:create-demo-data.sql'
+      url: jdbc:h2:file:./SalesDB;DB_CLOSE_DELAY=-1
```

Comment:
Can we not use an in memory db? Also can we change the name?

Reply:
yea name can be demoDB better

Reply:
I can change the name but using memory will need more work. I feel we could do with file based since its internally used and the actual management is through liquibase changelog.